src/DataService/Player/Player.php

```diff
@@ -0,0 +1,2460 @@
+     */
+    protected $compatibilityMode;
+
+    /**
+     * XML layout content for the player control rack.
+     *
+     * @var string
+     */
+    protected $controlLayoutXml;
+
+    /**
+     * Height of the player control rack.
+     *
+     * @var int
+     */
+    protected $controlRackHeight;
+
+    /**
+     * Custom CSS content for the player.
+     *
+     * @var string
+     */
+    protected $customCss;
+
+    /**
+     * URLs to remote custom CSS content for the player.
+     *
+     * @var \Psr\Http\Message\UriInterface[]
+     */
+    protected $customCssUrls;
+
+    /**
+     * Custom HTML content for the player (not currently used).
+     *
+     * @var string
+     */
+    protected $customHtml;
+
+    /**
+     * Custom JavaScript content for the player.
+     *
+     * @var string
+     */
+    protected $customJavaScript;
+
+    /**
+     * URLs to custom JavaScript content for the player.
+     *
+     * @var \Psr\Http\Message\UriInterface[]
+     */
+    protected $customJavaScriptUrls;
+
+    /**
+     * Additional attributes to include in the player HTML.
+     *
+     * @var array<string, string>
+     */
+    protected $customProperties;
+
+    /**
+     * The description of this object.
+     *
+     * @var string
+     */
+    protected $description;
+
+    /**
+     * Whether is player is available for customer retrieval.
+     *
+     * @var bool
+     */
+    protected $disabled;
+
+    /**
+     * The identifier for the advertising policy to use when the player is embedded in another site.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $embedAdPolicyId;
+
+    /**
+     * Indicates whether embedded players will be allowed to be viewed in fullscreen mode.
+     *
+     * @var bool
+     */
+    protected $embedAllowFullScreen;
+
+    /**
+     * The default height of the player when embedded in another site.
+     *
+     * @var int
+     */
+    protected $embedHeight;
+
+    /**
+     * The identifier for the restriction to apply to this player when embedded in another site.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $embedRestrictionId;
+
+    /**
+     * The default width of the player when embedded in another site.
+     *
+     * @var int
+     */
+    protected $embedWidth;
+
+    /**
+     * The set of plug-ins for this player.
+     *
+     * @var PlugInInstance[]
+     */
+    protected $enabledPlugIns;
+
+    /**
+     * Indicates if the player responds to commands from an IFrame parent element.
+     *
+     * @var bool
+     */
+    protected $enableExternalController;
+
+    /**
+     * The URL of the feed that will populate the related items list end card.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $endCardFeedUrl;
+
+    /**
+     * The custom failover HTML.
+     *
+     * @var string
+     */
+    protected $failoverHtml;
+
+    /**
+     * Reserved for future use. The default value is null.
```

Comment:
Same here, missing the real description of the property. Now I remember we had one like this in Media class too.